app/models/host.rb

```diff
@@ -11,6 +11,8 @@ class Host < ApplicationRecord
   scoped_search on: :has_test_results, ext_method: 'test_results?',
                 only_explicit: true, operators: ['=']
   scoped_search relation: :test_results, on: :profile_id
+  scoped_search on: :policy_id, ext_method: 'filter_by_policy',
+                only_explicit: true, operators: ['=']
```

Comment:
Does `scoped_search relation: :policy` not work here?

Reply:
It should work, but it would be only limited to lookup by a policy id. Note, that the policy id is not exposed in the APIs. Therefore, there is also a lookup by any of the profile ids that are assigned to the policy.

Reply:
Oh! That makes sense :) Thanks for the clarification